architecture/additional_concepts/authentication.adoc

```diff
@@ -135,23 +135,29 @@ To register additional clients:
 
 ----
 $ oc create -f <(echo '
-{
-  "kind": "OAuthClient",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "demo" <1>
-  },
-  "secret": "...", <2>
-  "redirectURIs": [
-    "http://www.example.com/" <3>
-  ]
-}')
+kind: OAuthClient,
+apiVersion: v1,
+metadata:
+  name: demo <1>
+secret: "...", <2>
+redirectURIs:
+   - http://www.example.com/ <3>
+grantStrategy: "...", <4>
+')
 ----
 <1> The `name` of the OAuth client is used as the `client_id` parameter when making requests to `_<master>_/oauth/authorize` and `_<master>_/oauth/token`.
 <2> The `secret` is used as the `client_secret` parameter when making requests to `_<master>_/oauth/token`.
 <3> The `redirect_uri` parameter specified in requests to `_<master>_/oauth/authorize` and `_<master>_/oauth/token` must be equal to (or prefixed by) one of the URIs in `redirectURIs`.
+<4> The `grantStrategy` is used to determine what action to take when this client requests tokens and has not yet been granted access by the user.
 ====
 
+Valid values for `*grantStrategy*` are:
```

Comment:
@adellape is this the best way to present these values here? Should we link the two tables together instead of duplicating them?